src/lib/ipsec/esp.lua

```diff
@@ -168,10 +162,59 @@ function esp_v6_decrypt:decapsulate (p)
                    "flow_id=" .. tostring(self.ip:flow_label()) .. ", " ..
                    "reason='" .. reason .. "'";
       logger:log("Rejecting packet ("..info..")")
-      return false
+      -- RFC4303 is somewhat unclear on a) whether or not packets that look replayed are eligible to trigger
+      -- recyns, and b) what 'consecutive' means in "number of consecutive packets that fail authentication":
+      -- Consecutive in time? Consecutive in their seq_low's?
+      -- Assuming a) should be 'yes' since the to-be-resynced packets might have seq_low's that just happen
+      -- to fall inside the replay window (seq_lo-wise) and would look replayed, yet aren't.
+      -- Assuming b) means 'in time', since systematic loss could stall resync indefinitely.
```

Comment:
Do we know how other implementations interpret the RFC? If not, could you compare with a mature implementation? OpenBSD?